addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10858,7 +10858,7 @@ msgstr ""
 #. label for 'pvr configuration incomplete' information dialog text
 #: xbmc/pvr/windows/GUIWindowPVRBase.cpp
 msgctxt "#19272"
-msgid "You need a tuner, backend software, and an add-on for the backend to be able to use PVR. Please visit http://kodi.wiki/view/PVR to learn more."
+msgid "You need to install, enable and configure a PVR add-on to use PVR. Please visit https://kodi.wiki/view/PVR to learn more."
```

Comment:
My english really not good, only maybe leave the PVR on end away?
```po
msgid "You need to install, enable and configure a PVR add-on to use. Please visit https://kodi.wiki/view/PVR to learn more."
```
Looks strange PVR and 3 words later again PVR.

Reply:
@jjd-uk do you have a suggestion how to improve the wording?

Reply:
I would suggest:

"To use PVR you need to install, enable and configure a PVR add-on. Please visit https://kodi.wiki/view/PVR to learn more."

Reply:
Thanks, will change the string accordingly, @jjd-uk

Reply:
done